src/client/components/OrganizationPanel.js

```diff
@@ -1,24 +1,64 @@
-import React from 'react'
-import styled from 'styled-components'
-import { Text } from '@aragon/ui'
+import React, { useContext } from 'react'
 import CardWrapper from '../wrappers/styleWrappers/CardWrapper'
-
-const addMore = () => console.log('add more orgs')
+import { useProfile } from '../hooks'
+import { ModalContext } from '../wrappers/modal'
+import OrganizationTile from './OrganizationTile'
+import { open, removeItem } from '../stateManagers/modal'
+import { Text, theme } from '@aragon/ui'
+import styled from 'styled-components'
 
 const OrganizationPanel = () => {
+  const { organizations, viewMode } = useProfile()
+  const { dispatchModal } = useContext(ModalContext)
+
+  const organizationsNotEmpty = Object.keys(organizations).length > 0
+
+  const cardProps = {
+    title: 'Organizations',
+    addMore: organizationsNotEmpty
+      ? () => dispatchModal(open('organization'))
+      : null,
+    addSeparators: true,
+    viewMode,
+  }
+
   return (
-    <CardWrapper title="Organisations" addMore={addMore}>
-      <Center>
-        <Text color="grey">Coming Soon</Text>
-      </Center>
+    <CardWrapper {...cardProps}>
```

Comment:
I'm not sure how useful it is to put `cardProps` in a temporary variable, since it's only used in this one spot. May as well put the props inline, here.

Reply:
True. As far as I remember it evolved from something useful to this. But it is also slightly out of scope for this PR - let's leave for future refactoring?